include/Object/entityComponentSystem.h

```diff
@@ -255,6 +290,12 @@ namespace Hop::Object
         int lua_getColour(lua_State * lua);
         int lua_setColour(lua_State * lua);
 
+        int lua_setTextureRegion(lua_State * lua);
```

Comment:
test these in new demo

Reply:
[ ] - demo doing loading via an update
[ ] - getting 
[ ] - setting
[ ] - drawing